### PR TITLE
Docs/UI: User manual modifications

### PR DESCRIPTION
**Pull request by brandelune**
This PR covers all the recent documentation modifications / improvements requests and suggestions that are not related to the new features documentation issues.

### CODE REVIEW

**Review by kosivantsov — APPROVED** [review]
Looks good to me.

**Review by Kazephil — COMMENTED** [review]
@brandelune 
Looks pretty good. I just sent a PR to your fork with suggested rephrasings and a rebase on the lastest matter.

**Review by Kazephil — APPROVED** [review]
Looks great now!

**Comment by brandelune** [comment]
This: [10 shortcuts to remember](https://github.com/omegat-org/omegat/pull/456/commits/090dd2c807bc58abd6d49555a2a7a4e57ac2761d) is a small table/hint/reminder for a few exotic shortcuts.

It is not filled yet (at the time of this writing).

**Comment by miurahr** [comment]
@brandelune  Is it any pragmatic reason we bundle `src/org/omegat/gui/editor/keys.md` in OmegaT.jar file?
Why not put it in `docs` or `docs_devel` ?

**Comment by miurahr** [comment]
merged into master.
Further tweaks can be in another PR.

**Comment by brandelune** [comment]
I seem to remember that Alex put it there to make sure developers are aware of default key functions. It would be better in docs_devel online as a `.md` file I think.

File: doc_src/en/HowTo_RestoreYourData.xml [hunk 1]
```diff
@@ -139,7 +139,51 @@
...
+		<para>Identify the project folder that OmegaT refuses to open and open
+		it in your file manager. The folder contents should be similar to a
+		standard <link linkend="chapter.project.folder">project</link>
+		folder.</para>
+      </listitem>
+
+	  <listitem>
+		<para>In the folder, you should find a
+		<filename>omegat.project</filename> file as well as a
+		<filename>omegat.project.bak</filename> file. Make sure that the files
```
Comment (miurahr) [inline doc_src/en/HowTo_RestoreYourData.xml:164]: https://github.com/omegat-org/omegat/commit/125ed163d9c20cf25de58bdd95c004b5b1741a59

Most recent OmegaT nightly does not produce `omegat.project.bak` but produce `omegat.project.YYYYMMDD.bak` and reuse it.

A change is proposed in July, 2021, and updated in Sep, 2022 then merged into master in Jan. 2023 at #415

Comment (brandelune) [reply doc_src/en/HowTo_RestoreYourData.xml:164]: Please create a new issue on GH for documentation of the code modifications.

Comment (brandelune) [reply doc_src/en/HowTo_RestoreYourData.xml:164]: I just created a new project and the backup file is not created. Can you check on your side?

**Pull request closed without merging**